src/frontend/src/components/Post/Post.jsx

```diff
@@ -144,12 +152,11 @@ const Post = ({ postUrl }) => {
       <ListSubheader className={classes.header}>
         <AdminButtons />
         <Typography variant="h1" title={post.title} id={post.id} className={classes.title}>
-          <span onClick={expandTitleOrNot} style={{ cursor: 'pointer' }}>
-            {expandToggle ? (
-              <span style={{ 'white-space': 'nowrap' }}>{post.title}</span>
-            ) : (
-              <span style={{ 'white-space': 'normal' }}>{post.title}</span>
-            )}
+          <span
+            onClick={toggleExpandHeader}
+            className={expandHeader ? classes.collapseHeader : classes.expandHeader}
```

Comment:
This is backwards.  If `expandHeader` is true then you should `expandHeader`.  See my comment above about default value.